config/crd/bases/cluster.x-k8s.io_kubeadmcontrolplanes.yaml

```diff
@@ -18,7 +18,6 @@ spec:
     shortNames:
     - kcp
     singular: kubeadmcontrolplane
-  preserveUnknownFields: false
```

Comment:
We likely don't want to drop this, it may require updating the kubebuilder annotation for KubeadmControlPlane to set `crd:preserveUnknownFields=false` as per https://github.com/kubernetes-sigs/controller-tools/pull/362